commands/next_lint_error.py

```diff
@@ -57,16 +57,25 @@ def _harvest_next(self) -> int:
             self.window.active_view().sel()[0].begin()
         )
         lines = set([])
+        next_change = None
         vid = self.window.active_view().id()
         for error_type in ['ERRORS', 'WARNINGS', 'VIOLATIONS']:
             for line, _ in ANACONDA[error_type].get(vid, {}).items():
                 lines.add(int(line))
 
-        lines = set(sorted(lines))
+        lines = list(set(sorted(lines)))
```

Comment:
With no `next_change` we don't need this cast as the index is used only once in the comprehension list